var/spack/repos/builtin/packages/gtkplus/package.py

```diff
@@ -88,6 +88,8 @@ class Gtkplus(AutotoolsPackage, MesonPackage):
     depends_on("cups", when="+cups")
     depends_on("libxfixes", when="@:2")
 
+    conflicts("%gcc@14:", when="@:3.24.29")
```

Comment:
```suggestion
    conflicts("%gcc@14:", when="@:3.24.35")
```
https://gitlab.gnome.org/GNOME/gtk/-/commit/a3e82f20a1a1e7d8ffbc960a8bb4aa65b0d151ee